install/common-pre-install/install_csm_services-cpi.md

```diff
@@ -0,0 +1,204 @@
+# Install CSM Services

```

Comment:
compared with actual csm docs, casm common installer is not adding any additional step. only we are adding expected output for few commands(that can be removed) also there is one known issue, will try to fix that using an automation script .
 I think we can refer to main doc itself.
As it will be difficult to sync the changes.

Reply:
Deleted the install-csm-services-cpi, and prepare-site-init-cpi md files and changed the links to the common MD files as per the discussion.